config/src/comments.rs

```diff
@@ -210,6 +210,14 @@ fn comments() -> HashMap<String, String> {
 		.to_string(),
 	);
 
+	retval.insert(
+		"always_stem_local".to_string(),
+		"
+#always stem local txs regardless of stem/fluff epoch (as per Dandelion++ paper)
```

Comment:
"local txs" sounds a bit weird, is this term used in the dandelion paper? Maybe "always_stem_ours" as a config option and "#always stem our own txs..." as a comment?

Reply:
Yeah good idea - will update.